src/app/app.constants.ts

```diff
@@ -30,29 +31,109 @@ const FooterDataNavigation = {
   ]
 };
 
-const navItems = [{
-    text: 'Case list',
-    href: '/cases',
-    active: false
-  }, {
-    text: 'Create case',
-    href: '/cases/case-filter',
-    active: false
-}];
+const applicationUserThemes: Theme[] =  [
+  {
+    roles: [
+      'caseworker-sscs-judge',
+      'caseworker-sscs-panelmember',
+      'caseworker-cmc-judge',
+      'caseworker-divorce-judge',
+      'caseworker-divorce-financialremedy-judiciary',
+      'caseworker-probate-judge',
+      'caseworker-ia-iacjudge',
+      'caseworker-publiclaw-judiciary',
+    ],
```

Comment:
roles hard coded? possible to move to LD or config?

Reply:
It is in the LD Config, this is the fall over, just in case LD stops working.